backend/engine/plugins/ghas_secrets/formatter.py

```diff
@@ -0,0 +1,127 @@
+from typing import Any
+from artemislib.github.api import GitHubAPI
+from artemislib.logging import Logger
+
+LOG = Logger("ghas_secrets")
+
+
+valid_types = {
+    "commit": True,
+    "wiki_commit": True,
+    "issue_title": True,
+    "issue_body": True,
+    "issue_comment": True,
+    "discussion_title": True,
+    "discussion_body": True,
+    "discussion_comment": True,
+    "pull_request_title": True,
+    "pull_request_body": True,
+    "pull_request_comment": True,
+    "pull_request_review": True,
+    "pull_request_review_comment": True,
+}
+
+
+def format_secret(
+    github: GitHubAPI, item_id: str, location: dict, alert: dict, author: str, author_timestamp: str
+) -> Any:
```

Comment:
Looks like this always returns a `dict`, so we can return that instead of `Any`.  Otherwise, we'll need to check the return with `isinstance()` at the callsite.